Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -537,6 +545,17 @@ define([
         var boundingVolume = defaultValue(surfaceTile.orientedBoundingBox, surfaceTile.boundingSphere3D);
 
         if (frameState.mode !== SceneMode.SCENE3D) {
+
+            // Check if the tile is outside the limit area in 2D
```

Comment:
This is the coarse-grained CPU check, right?  Then there is a fine-grained GLSL check like 3D Tiles clipping planes?  Might want to add a tad more doc here.

Also - is it possible and a good idea to only modify the GLSL for tiles that might intersect the boundary - so tiles completely outside are culled on the CPU and tiles completely inside would not have their GLSL modified?  Are you already doing this?

Reply:
> tiles completely outside are culled on the CPU and tiles completely inside would not have their GLSL modified? Are you already doing this?

Yes, convinced myself this is working by hopping into `GlobeSurfaceShaderSet` and setting debug break points around the `clippedByBoundaries` stuff, then zooming in and out.
I'll add the extra doc.